cns/middlewares/k8sSwiftV2.go

```diff
@@ -182,6 +182,11 @@ func (k *K8sSWIFTv2Middleware) getIPConfig(ctx context.Context, podInfo cns.PodI
 			if prefixSize != prefixLength {
 				return nil, errors.Wrapf(errInvalidMTPNCPrefixLength, "mtpnc primaryIP prefix length is %d", prefixSize)
 			}
+			// Parse MTPNC SubnetAddressSpace to get the subnet prefix length
+			subnet, subnetPrefix, err := utils.ParseIPAndPrefix(interfaceInfo.PrimaryIP)
```

Comment:
lgtm, can we add an UT to make sure this parses the information correctly?

Reply:
added subnetAddressSpace in mockclient to be covered by existing UTs